### ISSUE TITLE
Java 8 in the parent pom

### ISSUE THREAD

**Issue by optimatika** [ujmp/universal-java-matrix-package#2]
In the develop branch you need to set the Java version to 1.8 otherwise you get some very strange error.

**Comment by holger-arndt** [comment]
Yes, I've seen that ojAlgo 38.1 does not compile with Java 7 in the development branch.Unfortunately I cannot switch to Java 8 because we still have some Debian servers running with the older version. So I assume the only option is to downgrade to 37.1?

**Comment by optimatika** [comment]
In that case yes, v37.1 is the one for you.

**Comment by VestOfHolding** [comment]
In the years since, can there be another crack at updating Java to 8 or even 11?
